commit 1a05ea6e5e42ec8a4270cb8a34fc9548d81976ca
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Thu Jun 14 11:22:17 2018 +0100

    Revert "dwc_otg: Disable fiq by default until it is fixed"
    
    This reverts commit fbe0a4908cd0afbe6ba54526103538f833d2d68b.

diff --git a/drivers/usb/host/dwc_otg/dwc_otg_driver.c b/drivers/usb/host/dwc_otg/dwc_otg_driver.c
index 315f260fff48..e945900c503c 100644
--- a/drivers/usb/host/dwc_otg/dwc_otg_driver.c
+++ b/drivers/usb/host/dwc_otg/dwc_otg_driver.c
@@ -241,9 +241,9 @@ static struct dwc_otg_driver_module_params dwc_otg_module_params = {
 };
 
 //Global variable to switch the fiq fix on or off
-bool fiq_enable = false;
+bool fiq_enable = 1;
 // Global variable to enable the split transaction fix
-bool fiq_fsm_enable = false;
+bool fiq_fsm_enable = true;
 //Bulk split-transaction NAK holdoff in microframes
 uint16_t nak_holdoff = 8;
 
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1166_1a05ea6e5e42ec8a4270cb8a34fc9548d81976ca.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1166_1a05ea6e5e42ec8a4270cb8a34fc9548d81976ca.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
